src/SIL.XForge.Scripture/ClientApp/src/app/change-password/change-password.component.ts

```diff
@@ -0,0 +1,60 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { SFProjectService } from '../core/sfproject.service';
+
+@Component({
+  selector: 'app-change-password',
+  templateUrl: './change-password.component.html',
+  styleUrls: ['./change-password.component.scss']
+})
+export class ChangePasswordComponent implements OnInit {
+  changePasswordForm: FormGroup;
+  isSubmitted = false;
+  btnChangePasswordEnabled: Boolean = true;
```

Comment:
I would just use `boolean` as the type instead of `Boolean` to be consist with existing code.